app-admin/qtpass/qtpass-1.3.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="QtPass-${PV}"
```

Comment:
it's used only once, no need to declare a variable then.

Reply:
Removed MY_P.